hack/verify-all.sh

```diff
@@ -6,10 +6,16 @@ set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-bash "$REPO_ROOT/hack/verify-codegen.sh"
-bash "$REPO_ROOT/hack/verify-crdgen.sh"
-bash "$REPO_ROOT/hack/verify-staticcheck.sh"
+# Show progress
+set -x
+
+# Orders are determined by two factors:
+# (1) Less Execution time item should be executed first.
+# (2) More likely to fail item should be executed first.
+bash "$REPO_ROOT/hack/verify-lifted.sh"
 bash "$REPO_ROOT/hack/verify-import-aliases.sh"
+bash "$REPO_ROOT/hack/verify-staticcheck.sh"
 bash "$REPO_ROOT/hack/verify-vendor.sh"
 bash "$REPO_ROOT/hack/verify-swagger-docs.sh"
-bash "$REPO_ROOT/hack/verify-lifted.sh"
+bash "$REPO_ROOT/hack/verify-crdgen.sh"
+bash "$REPO_ROOT/hack/verify-codegen.sh"
```

Comment:
I remember `swagger-docs` is dependent on `codegen`, it shall move after the later.
Is it @RainbowMango?

Reply:
> I remember swagger-docs is dependent on codegen

Yes. I have lesson learned before and restricted the order in `hack/update-all.sh`:
https://github.com/karmada-io/karmada/blob/141690eb9bf26c1286f3baba358d7633bf5e52ad/hack/update-all.sh#L9-L16

> it shall move after the later.

But for `hack/verify-all.sh`, which is just verify and test, we can assume all generated files are ready, so the dependency order doesn't matter.